build/target-repository/docs/rector_rules_overview.md

```diff
@@ -208,6 +208,41 @@ return static function (ContainerConfigurator $containerConfigurator): void {
 
 <br>
 
+### FunctionArgumentDefaultValueReplacerRector
+
+Streamline the operator arguments of version_compare function
+
+:wrench: **configure it!**
+
+- class: [`Rector\Arguments\Rector\FuncCall\FunctionArgumentDefaultValueReplacerRector`](../../../rules/Arguments/Rector/FuncCall/FunctionArgumentDefaultValueReplacerRector.php)
```

Comment:
this `../../../` should be fixed in https://github.com/rectorphp/rector-src/pull/92